src/components/MonthlyGood/MonthlyGoodDropInPaymentWrapper.vue

```diff
@@ -105,60 +107,107 @@ export default {
 				});
 		},
 		doBraintreeMonthlyGood(nonce, paymentType) {
-			// Apollo call to the query mutation
-			this.apollo.mutate({
-				mutation: braintreeCreateMonthlyGoodSubscription,
-				variables: {
-					paymentMethodNonce: nonce,
-					amount: numeral(this.amount).format('0.00'),
-					donateAmount: numeral(this.donateAmount).format('0.00'),
-					dayOfMonth: numeral(this.dayOfMonth).value(),
-					category: this.category,
-					isOnetime: this.isOneTime
-				}
-			}).then(kivaBraintreeResponse => {
-				// Check for errors in transaction
-				if (kivaBraintreeResponse.errors) {
-					const errorCode = _get(kivaBraintreeResponse, 'errors[0].code');
-					const errorMessage = _get(kivaBraintreeResponse, 'errors[0].message');
-					const standardErrorCode = `(Braintree error: ${errorCode})`;
-					const standardError = `There was an error processing your payment.
+			if (this.action === 'Update') {
+				// Apollo call to the query mutation
+				this.apollo.mutate({
+					mutation: braintreeUpdateSubscriptionPaymentMethod,
+					variables: {
+						paymentMethodNonce: nonce,
```

Comment:
what are the other properties we're updating?

Reply:
The other properties are handled outside of the braintree drop in. If you recall, the modal has an "update settings" section and an "update payment method" section. Each section is independent of each other with it's own submit button.